packages/backend/server/src/core/auth/controller.ts

```diff
@@ -30,6 +32,9 @@ import { AuthService } from './service';
 import { CurrentUser, Session } from './session';
 import { TokenService, TokenType } from './token';
 
+// set DNS servers
+setServers(['1.1.1.1', '8.8.8.8']);
```

Comment:
why we need this? Our Node.js instances are in k8s cluster, there may rpc requires `coredns` in the future, can we remove this?

Reply:
in the local dev env, if we enable network software like clash tun/surge/tailscale/zerotier, thire DNS hijacking logic may not correctly provide domain records other than DNS. let me add a conditional logic

Reply:
fixed